charts/nginz/templates/conf/_deeplink.html.tpl

```diff
@@ -3,10 +3,18 @@
      (or search for "deeplink" on docs.wire.com)
      for details on use of the deeplink*/}}
 <html>
-  <head></head>
-  <body>
+  <head>
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body style="font-size: 1.5em; font-family: sans">
+    <h4>Wire client setup</h4>
     {{- if hasKey .Values.nginx_conf "deeplink" }}
-    <a href="wire://access/?config={{ .Values.nginx_conf.deeplink.endpoints.backendURL }}/deeplink.json">Click here for access</a>
+    <p>
+    <label>backend: <code>{{ .Values.nginx_conf.deeplink.endpoints.backendURL }}/deeplink.json</code></label>
+    </p>
+    <a href="wire://access/?config={{ .Values.nginx_conf.deeplink.endpoints.backendURL }}/deeplink.json">
+    <button >Configure client to use this backend</button>
```

Comment:
I'm wondering if it wouldn't be better to keep this as text link as one can easily copy them (right click and send to coworker...) :thinking:

Reply:
`<marquee>Configure client to use this backend</marquee>`